crates/oxc_semantic/src/symbol.rs

```diff
@@ -54,6 +54,10 @@ impl SymbolTable {
         self.flags[symbol_id]
     }
 
+    pub fn set_flag(&mut self, symbol_id: SymbolId, includes: SymbolFlags) {
+        self.flags[symbol_id] |= includes;
```

Comment:
one last nitpick: this is unioning the flag, which means the function name and documentation should indicate this.

Reply:
Good point! Thanks!